### ISSUE TITLE
[Doc] Lack of understanding "Namespace change events" for Zookeeper

### ISSUE THREAD

**Issue by gulecroc** [apache/pulsar#23748]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### What issue do you find in Pulsar docs?

It is written that using namespace change event :

> * Avoid using ZooKeeper and introduce more loads to ZooKeeper.

 https://pulsar.apache.org/docs/4.0.x/concepts-multi-tenancy/#namespace-change-events-and-topic-level-policies

I don't understand the purpose of this : is it use more or less Zookeeper ?

Best regards,
Guillaume

### What is your suggestion?

Correct description

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by lhotari** [comment]
> I don't understand the purpose of this : is it use more or less Zookeeper ?

I agree, it's hard to understand. There's a need to improve this description. My assumption is what it means is that namespace change events (and topic level policies) provide a way to reduce load on the metadata store backend (which is Zookeeper in most cases with Pulsar).
